includes/class-wc-calypso-bridge-base-admin-menu.php

```diff
@@ -0,0 +1,396 @@
+<?php
+
+use Automattic\Jetpack\Status;
```

Comment:
Is this dependency on Jetpack required? Try to remove if you can.

Reply:
Oh yes! I forgot to remove it